### DISCUSSION TITLE
Is it possible to "bootstrap" a backup?

### DISCUSSION THREAD

**Discussion by wsw70** [borgbackup/borg#6925] [Q&A] [answered]
I will backup a folder that has some voluminous subfolders. The backup is remote so the first pass takes an awful amount of time.

**Is there a way to "bootstrap" the backup by manually planting the required files?** The raw transfer is fine (I can copy either the files or "something out of the files" in a reasonable time)

The workaround I have in mind, for now, is to copy an existing backup as-it, change the ID, and hope that the deduplicating will kick in.

**Comment by ThomasWaldmann** [comment, marked as answer]
You can do the first backup locally and then transfer the resulting repo directory and its contents to the final place.

OTOH, you can also just wait. If there is some interruption, just restart `borg create` until it completes. At some time, all will be transferred and then it'll get much faster.

**Reply by jdchristensen** [reply]
You can also backup subfolders of the data initially (i.e. partition the data into small subsets).  Then, when you do a backup of the complete set, most of the chunks will already be there.

**Reply by wsw70** [reply]
> You can do the first backup locally and then transfer the resulting repo directory and its contents to the final place.

I already have a long-term backup (with daily, weekly, and monthly versions) so I would somehow need to extract the latest backup. Or (and this is what I will probably do) - start a new backup to have only one version and copy that.

> OTOH, you can also just wait.

This is what I am going to do ultimately - for now I want to have a remote backup of the essential directories, and I will have it backup the less essential one over a week.
